public/index.html

```diff
@@ -438,16 +438,25 @@ <h1>Match Statistics</h1>
 				    </div>
 			</div>
         </div>
-        
+
         <div ng-controller="netballCtrl" class="netball left top scale-fade" ng-show="netball.show">
-            <div class="row">    	
+            <div class="row">
                 <div class="score lanc"><span ng-show="netball.showcurrent"><span ng-show="showcurrentlancs">&diams;</span></span>LANCS {{netball.lancScore}}</div>
                 <div class="score york">{{netball.yorkScore}} YORK<span ng-show="netball.showcurrent"><span ng-show="showcurrentyork">&diams;</span></span></div>
                 <div class="clock">{{clock}}</div>
                 <div class="netballquarter fade" ng-show="netball.showquarter"><h1>{{netball.currQuarter}}</h1></div>
             </div>
         </div>
 
+        <div ng-controller="waterpoloCtrl" class="waterpolo left top scale-fade" ng-show="waterpolo.show">
+            <div class="row">
+                <div class="score lanc"><span ng-show="waterpolo.showcurrent"><span ng-show="showcurrentlancs">&diams;</span></span>LANCS {{waterpolo.lancScore}}</div>
+                <div class="score york">{{waterpolo.yorkScore}} YORK<span ng-show="waterpolo.showcurrent"><span ng-show="showcurrentyork">&diams;</span></span></div>
+                <div class="clock">{{clock}}</div>
+                <div class="netballquarter fade" ng-show="waterpolo.showquarter"><h1>{{waterpolo.currentQuarter}}</h1></div>
```

Comment:
Why you using the netball quarter style class here instead of a specific waterpolo one?